hw/top_earlgrey/data/ip/chip_sysrst_ctrl_testplan.hjson

```diff
@@ -76,6 +80,7 @@
         si_stage:SV3
         lc_states: ["DEV", "PROD", "PROD_END", "RMA"]
         tests: ["chip_sw_sysrst_ctrl_ulp_z3_wakeup"]
+        bazel: ["//sw/device/tests:pwrmgr_random_sleep_all_wake_ups"]
```

Comment:
This test uses the regular KEY_INTR wakeup, not ULP, so it is not a good match to this testpoint.

By the way, in line 78 above there is a typo in ULATRA. I guess we don't check for legal features.

Reply:
You are right, I will port the `sysrst_ctrl_ulp_z3_wakeup_test` test to sival instead.

Unfortunately we do not check for legal features but I believe that there was a plan to implement that.